specs/windows/ntfs_acl_permissions.table

```diff
@@ -0,0 +1,10 @@
+table_name("ntfs_acl_permissions")
+description("Retrieve NTFS ACL permission information for files and directories.")
+schema([
+  Column("path", TEXT, "Path to the file or directory."),
+  Column("type", TEXT, "Drive letter of the encrypted drive."),
+  Column("principal", TEXT, "Persistent ID of the drive."),
+  Column("access", TEXT, "Persistent ID of the drive."),
+  Column("inherited_from", TEXT, "The bitlocker conversion status of the drive."),
+])
+implementation("system/windows/ntfs_acl_permissions@genNTFSACLPerms")
```

Comment:
I don't _think_ that you need the `system/windows` here, as the impl file should get picked up correctly.

Reply:
Gonna leave this in just to be safe and also to make it easier for any readers to easily locate the implementation 😉